### DISCUSSION TITLE
Raise Governor Limit on Klaytn

### DISCUSSION THREAD

**Discussion by steve-jeon-KF** [wormhole-foundation/wormhole#3785] [Governance]
The governor limit on Wormhole for Klaytn is currently set to $500,000 as for now. We are planning to initiate [Onchain Boosting](https://govforum.klaytn.foundation/t/notice-boosting-onchain-metrics-pre-d2i/759) soon and use Wormhole as the main bridge at this time, which is related to the Dragon merge and the [new DeFi initiative](https://govforum.klaytn.foundation/t/kgp-26-dragon-defi-initiative-d2i-crafting-the-core-of-project-dragon-s-defi-ecosystem/730). However, there is a daily limit on the Wormhole bridge. Therefore, we would like to avoid a situation where transactions are rate-limited due to low caps of the tokens when Wormhole native ETH and others are bridging. Raising caps is necessary for ensuring a smooth user experience via Wormhole and aligning it with other EVM networks. 

Proposed 24hr(Daily) Limit: $2,000,000
(Please apply the increased limit of $500K per transaction to the big transaction limit.)

**Comment by SimonKraus** [comment]
Awesome to see the Onchain Boosting Initiative that will open up great opportunities for the community and serves the right incentives to bridge over. For sure Staking Facilities is signalling support for increasing the governor limits awaiting the traffic of the campaign 💪 🚀

**Reply by steve-jeon-KF** [reply]
Thanks for your comment @SimonKraus! We're thrilled to see Staking Facilities on board, supporting the Onchain Boosting Initiative. Our Onchain Boosting Initiative is scheduled to take place early next week, and we anticipate at least over a million dollars' worth of assets being bridged by liquidity providers starting next week. Looking forward to the governor limit increment soon. 🚀
